src/ui/settings/ViewTipsAndTricks/ViewTipsAndTricks.qml

```diff
@@ -0,0 +1,166 @@
+        anchors.left: parent.left
+        anchors.right: parent.right
+        anchors.topMargin: VPNTheme.theme.vSpacing
+        anchors.leftMargin: VPNTheme.theme.windowMargin
+        anchors.rightMargin: VPNTheme.theme.windowMargin
+
+        spacing: 0
+
+        //superfluous columnlayout that fixes a binding loop
+        ColumnLayout {
+            anchors.left: parent.left
+            anchors.right: parent.right
+
+            spacing: 0
+
+            // Highlighted tutorials
+            Flow {
+                Layout.fillWidth: true
+                spacing: VPNTheme.theme.vSpacingSmall
+
+                Loader {
+                    id: highlightedTutorialLoader
+                    property variant highlightedTutorial: VPNAddonManager.pick(addon => addon.type === "tutorial" && addon.highlighted)
+
+                    height: VPNTheme.theme.tutorialCardHeight
+                    width: vpnFlickable.width < VPNTheme.theme.tabletMinimumWidth ? parent.width : (parent.width - parent.spacing) / 2
+
+                    active: highlightedTutorial
+                    visible: active
+
+                    sourceComponent: VPNTutorialCard {
+                        objectName: "highlightedTutorial"
+                        width: parent.width
+                        height: parent.height
+
+                        imageSrc: highlightedTutorial.image
+                        title: qsTrId(highlightedTutorial.titleId)
+                        description: qsTrId(highlightedTutorial.subtitleId)
+
+                        onClicked: {
+                            VPNTutorial.play(highlightedTutorial);
+                            VPNCloseEventHandler.removeAllStackViews();
+                        }
+                    }
+                }
+
+                VPNTutorialCard {
+                    objectName: "featureTourCard"
+
+                    width: vpnFlickable.width < VPNTheme.theme.tabletMinimumWidth ? parent.width : (parent.width - parent.spacing) / 2
+                    height: VPNTheme.theme.tutorialCardHeight
+
+                    imageSrc: "qrc:/ui/resources/sparkling-check.svg"
+                    imageBgColor: "#2B2A33"
+                    title: VPNl18n.TipsAndTricksFeatureTourCardTitle
+                    description: VPNl18n.TipsAndTricksFeatureTourCardDescription
+                    onClicked: featureTourPopup.startTour();
+                }
+            }
+
+            // Quick Tips
+            VPNFilterProxyModel {
+                id: quickGuidesModel
+                source: VPNAddonManager
+                filterCallback: ({ addon }) => isQuickAddon(addon, "guide")
+            }
+
+            VPNFilterProxyModel {
+                id: quickTutorialsModel
+                source: VPNAddonManager
+                filterCallback: ({ addon }) => isQuickAddon(addon, "tutorial")
+            }
+
+            TipsAndTricksSection {
+                Layout.topMargin: 32
+
+                id: quicksTips
```

Comment:
the smallest of small nits: the [qml coding convetions](https://doc.qt.io/qt-6/qml-codingconventions.html) say to put the id property at the top of the object block

It is totally fine the way it is, I'm just trying to give you a warm welcome to qml 😄